requirements.txt

```diff
@@ -7,7 +7,7 @@
 backports.ssl-match-hostname==3.5.0.1  # via docker-py
 certifi==2019.6.16        # via requests
 chardet==3.0.4            # via requests
-click==6.6
+click==7.1.2
```

Comment:
I don't think it's related anyhow with the pipenv support, is it? If not, let's better update it separately